envs/example/swift/group_vars/all.yml

```diff
@@ -3,7 +3,7 @@ stack_env: example_swift
 swift_floating_ip: "{{ hostvars[groups['swiftnode'][0]][primary_interface]['ipv4']['address'] }}"
 
 undercloud_cidr: 
-  - 10.1.1.0/24
+  - cidr: 10.1.1.0/24
```

Comment:
In a previous PR it was determined that instead of trying to logic out what the gateway for a CIDR should be in the task, to just define the gateway in the same place that the CIDR is defined.